### PR TITLE
The Evolution Of Resource Sharing Technologies Over The Internet

### PR DESCRIPTION
**Pull request by KiokoM**
Contains index.md and hero image

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by KiokoM** [comment]
This fixes #977

**Comment by lalith1403** [comment]
Change the title of the pr to the title of the article.

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+![hero.jpg](/engineering-education/the-evolution-of-resource-sharing-technologies-over-the-internet/hero.jpg)
+
+With the growth of the internet and the web, there has always been a need for resource sharing and collaboration. We could work more effectively through collaborating with others. This often involved the sharing or interaction of various software products. There was a need for distributed computing. However, one of the biggest challenges of distributed computing was how to get software components located in different geographic locations to interact in a standardized way. Several technologies were created to try and meet this need.
+
+### 1. [CORBA OMG](https://en.wikipedia.org/wiki/Common_Object_Request_Broker_Architecture)
```
Comment (lalith1403) [inline articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md:5]: Don't add the link here. Instead, add the link below.

File: articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+While the CORBA standard was very promising, several issues were encountered. The biggest of which was the poor implementation of the standard. Also, there was quite a bit of politics brought about by competing vendors, who tried to steer the standard in a particular direction. The standard was also a bit too complex.
+
+CORBA was built with the aim of:
+ - Improve the use of distributed systems.
+ - It is currently used in Object-oriented programming
+ - Allowing multiple programming languages to communicate with each other
+
+### 2. [DCOM](https://en.wikipedia.org/wiki/Distributed_Component_Object_Model)
```
Comment (lalith1403) [inline articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md:16]: Add the link below! Do the same for all headers.

File: articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    1. **Uniform Interface** - Requests from different clients should look the same. That is, the same resources should not have more than one URI.
+    2. **Client-Server Separation** - The client and the server should not act as one but should be independent, which means that their only form of interaction is through requests and responses.
+    3. **Statelessness** - There should not be any server-side sessions. A single request should contain all the information the server needs to know.
+    4. **Cacheable resources** - Server responses should contain information about whether the data they send is cacheable or not.
+    5. **Layered System** - There might be several layers of servers between the client and the server that returns the response. This should not affect either the request or the response.
+---
```
Comment (lalith1403) [inline articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md:47]: Conclude the article. Create a section called Conclusion, and summarize everything in the article.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written article. The concepts are explained in-depth and offer a lot of information concerning the internet. Very few minor changes are required.

File: articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md (hunk 3 above)
Comment (KiokoM) [reply articles/the-evolution-of-resource-sharing-technologies-over-the-internet/index.md:47]: Got it, I'll work on it and update. Thank you

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
@KiokoM  👋  Good afternoon - upon running your article through our 3rd party plagiarism checker it raised a some red flags and the % was higher than we typically like to see. 
If you could please see the attached PDF and revisit the article to make sure we are creating content that is unique and original. 

I did see the links and sources 👍  those are fantastic - but the majority of the article needs to be original content 🚀  

Thank you
[The Evolution Of Resource Sharing Technologies Over The Internet #1009.pdf](https://github.com/section-io/engineering-education/files/5793160/The.Evolution.Of.Resource.Sharing.Technologies.Over.The.Internet.1009.pdf)

**Comment by KiokoM** [comment]
@hectorkambow Hi, I went through the document and I'm currently working on making the article more original. Sorry for the delayed feedback but I'll have it ready today by end of day

Thank you.

**Comment by lalith1403** [comment]
@hectorkambow The author has made changes. Can you check once?

**Pull request merged**